lib/solid_angle.h

```diff
@@ -1,111 +0,0 @@
-#pragma once
-
-#include "mesh.h"
-
-#include <cmath>
-
-using Point = RadiosityMesh::Point;
-
-/**
- * Compute the solid angle subtended by triangle from `O`.
- *
- * See Eriksson, Folke (1990). "On the measure of solid angles". Math. Mag. 63
- * (3): 184–187.
- *
- * @param  O       origin
- * @param  A, B, C vertices of the triangle
- * @return         solid angle
- */
-float solid_angle(const Point& O, const Point& A, const Point& B,
```

Comment:
> Merge radiosity functions and tests.  …
> * Move solid angle functions and test to lib/radiosity.h.
> * Move implementations of form factor functions to header, s.t.
>   they can be inlined.

We are basically using this function only in radiosity, so moved them into the library file of radiosity.